src/ArmTemplates/Commands/Executors/CreatorExecutor.cs

```diff
@@ -1,6 +1,7 @@
 ﻿namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Commands.Executors
 {
-    internal class CreatorExecutor
+    class CreatorExecutor
     {
+        // TODO refactor creatorApplicationCommand like ExtractorExecutor is
```

Comment:
If you add a TODO, can you link to the GH issue that's tracking this please?

Reply:
That is a great suggestion, thank you. Added the issue #623 and saved link near TODO